### PR TITLE
Examples and tutorials

### PR DESCRIPTION
**Pull request by mashby2022**
## Description

Changes proposed in this pull request:

- adding examples and tutorials section to docs 
- 

## Testing

- run npm start 
- chose the unreleased version
- check the content and formatting

## Related issue(s)

https://github.com/kubeshop/botkube-cloud/issues/880

### CODE REVIEW

**Comment by madebyrogal** [comment]
@mashby2022 I can't fix "Dead link checker" cos I don't know the right links.
I can show you the ropes how to check it by your own.

To run "Dead link checker" locally run this command
`mage checkLinks` - if you will have error like: `unknown mage` then you can ask anybody of us to help you setup on your machine

For now you don't have to run this command at all because all errors are display in the details of "Dead link checker" details:
image

Of course if you need help just ping me or anybody from the dev team

File: docs/examples-and-tutorials/usecases/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## GitOps Integrations
+
+- [Flux Plugin for Day 2 Operations](https://botkube.io/blog/creating-the-botkube-flux-plugin-for-day-2-operations): Discover how Botkube's Flux plugin simplifies Day 2 operations and provides tailored solutions for various K8s management needs.
+- [ArgoCD Plugin for Optimizing Collaboration & Notifications](https://botkube.io/blog/optimizing-collaboration-and-notifications-with-the-botkube-argocd-plugin): Learn about how Botkube's engineering team using the ArgoCD plugin to optimize their deployment experience.
+- [Enhancing GitOps Workflows with Botkube](https://botkube.io/blog/enhancing-gitops-workflows-with-botkube): Explore Botkube's capabilities in optimizing GitOps workflows.
+
+## Automating the K8s Troubleshooting workflow
+
+- [Using Keptn as a Source](https://botkube.io/blog/implementing-your-own-botkube-plugin-a-real-life-use-case): Discover how to leveraging Keptn as a source to adapt to various troubleshooting scenarios.
```
Comment (madebyrogal) [inline docs/examples-and-tutorials/usecases/index.md:17]: Shouldn't be "... how to leverage Keptn ..."

File: docs/examples-and-tutorials/tutorials/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- [Kubectl Plugin Usage](https://botkube.io/learn/kubectl-cheat-sheet): Utilize Botkube's integration with Kubectl for executing Kubernetes commands conveniently.
+- [Open AI Integration](https://botkube.io/blog/doctor-plug-in-tutorial): Integrate Open AI with Botkube for intelligent automation and enhanced interactions within Kubernetes environments.
+- [Helm Plugin Setup](https://botkube.io/learn/helm-charts): Seamlessly manage Helm deployments and releases with Botkube's Helm integration.
+- [Flux Plugin Configuration](https://botkube.io/blog/streamlining-gitops-with-the-botkube-flux-plugin): Set up and configure Botkube's Flux plugin for automated deployment and synchronization of Kubernetes resources.
+- [Custom Executor Plugin Development](https://botkube.io/integration/custom-executor-plugin): Develop and integrate custom executor plugins with Botkube for tailored Kubernetes operations.
+
+### Source Plugins
+
+- [Kubernetes Events Monitoring](https://botkube.io/blog/five-essential-kubernetes-tasks: Monitor Kubernetes events effortlessly with Botkube's built-in source plugin.
```
Comment (madebyrogal) [inline docs/examples-and-tutorials/tutorials/index.md:33]: missing closing bracket
```
[Kubernetes Events Monitoring](https://botkube.io/blog/five-essential-kubernetes-tasks)

File: docs/examples-and-tutorials/videos/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- [ArgoCD Workflows](https://youtu.be/1i59PN-65SY?feature=shared)
+- [Zapier for Kubernetes](https://www.youtube.com/live/VJzSRTQpHhs?feature=shared)
+
+## Community calls
+
+- [Open Source Friday- ChatOps for Kubernetes](https://www.youtube.com/live/tdyPuMA2mww?feature=shared)
+- [CNCF Webinar- Unlocking the Power of Kubernetes Collaborative Troubleshooting](https://youtu.be/6VTEOOfIbIk?feature=shared)
+- [Tailwarden- ChatOps for Kubernetes](https://youtu.be/UOURPb03mxU?feature=shared)
+- [Kubefirst- Monitor & Act on Your Kubernetes Cluster](https://www.youtube.com/live/RLmU472lync?feature=shared)
+- [CNCF Islamabad- Unlocking K8s Troubleshooting Best Practices](https://www.youtube.com/live/Ry_UijS1ZNk?feature=shared)
```
Comment (madebyrogal) [inline docs/examples-and-tutorials/videos/index.md:27]: Fix spacing (should be space before word and dash)
[Open Source Friday - ChatOps for Kubernetes]

File: docs/examples-and-tutorials/videos/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [Cloud Lunch and Learn - K8s Troubleshooting Best Practices in Azure](https://www.youtube.com/live/x0TBA6TpfqU?feature=shared)
+- [Kunal Kushwaha- Collaborative Troubleshooting and Monitoring in K8s With Botkube](https://youtu.be/coDSM9LiOB4?feature=shared)
```
Comment (madebyrogal) [inline docs/examples-and-tutorials/videos/index.md:29]: Fix spacing (dash)

File: mage
Comment (madebyrogal) [inline mage:1]: remove this folder

File: wink:)
Comment (madebyrogal) [inline wink:):1]: Not sure what is this. Can you remove this file

**Review by madebyrogal — COMMENTED** [review]
Overall works fine just some small typos/comments

But when we finalised the main version we need to check 1.7, 1.8 and add this docs to 1.9
I've check 1.7 and 1.8 but they are not sync with unreleased but I would focus on them when we have finial version of this docs
Good work Maria!

File: docs/examples-and-tutorials/usecases/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- [ArgoCD Plugin for Optimizing Collaboration & Notifications](https://botkube.io/blog/optimizing-collaboration-and-notifications-with-the-botkube-argocd-plugin): Learn about how Botkube's engineering team using the ArgoCD plugin to optimize their deployment experience.
+- [Enhancing GitOps Workflows with Botkube](https://botkube.io/blog/enhancing-gitops-workflows-with-botkube): Explore Botkube's capabilities in optimizing GitOps workflows.
+
+## Automating the K8s Troubleshooting workflow
+
+- [Using Keptn as a Source](https://botkube.io/blog/implementing-your-own-botkube-plugin-a-real-life-use-case): Discover how to leverage Keptn as a source to adapt to various troubleshooting scenarios.
+- [Streamlining Helm Deployments](https://botkube.io/case-studies/civo): Learn how one of Botkube's users utilizes the Botkube Helm plugin to streamline their Helm deployments and updates.
+- [Github Issue Reporter for Failing K8s Apps](https://botkube.io/blog/build-a-github-issues-reporter-for-failing-kubernetes-apps-with-botkube-plugins): Automate the reporting of issues with your Kubernetes applications on GitHub directly from your Slack channel.
```
Comment (madebyrogal) [inline docs/examples-and-tutorials/usecases/index.md:19]: GitHub instead Github

File: package.json [hunk 6]
```diff
@@ -49,7 +49,7 @@
     "eslint": "^8.34.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "prettier": "^2.8.8",
```
Comment (madebyrogal) [inline package.json:52]: please revet this it should be 
"prettier": "^2.8.8",

**Review by mashby2022 — COMMENTED** [review]
Changed the Github typo and updated the prettier version

**Review by madebyrogal — APPROVED** [review]
Ok great! now we have it 🥇 
Please remember that we have one thing todo before merging it and after it will be approved also by Pawel.
We need to copy/past current examples-and-tutorials docs to older versions of docs.
We was talking to move it 2 version back.

**Review by pkosiec — COMMENTED** [review]
LGTM apart from some small comments and the fact the changes aren't synchronized to 1.9. Please apply the fixes first 👍

File: versioned_docs/version-1.7/examples-and-tutorials/_category_.json
[the remark this answers is not in the document]
Comment (madebyrogal) [reply versioned_docs/version-1.7/examples-and-tutorials/_category_.json:1]: @mashby2022 I will do that after you resolve all other comments

File: docs/examples-and-tutorials/_category_.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Examples and tutorials",
+  "position": 10
```
[the remark this answers is not in the document]
Comment (madebyrogal) [reply docs/examples-and-tutorials/_category_.json:3]: @mashby2022 I can help with that. Let me know

File: docs/configuration/index.mdx [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 id: configuration
 title: Configuration
-sidebar_position: 2
+sidebar_position: 3
```
Comment (pkosiec) [inline docs/configuration/index.mdx:4]: This is the same position as "Usage". @madebyrogal can you update the positions everywhere, and sync the changes to the previous versions?

**Review by pkosiec — APPROVED** [review]
LGTM with one comment

**Pull request merged**